### ISSUE TITLE
[Eval] Relay evaluation metrics/schema from the Impact Metrics Garden

### ISSUE THREAD

**Issue by bitbeckers** [hypercerts-org/hypercerts-api#47]

**Comment by holkexyz** [comment]
@bitbeckers would this be a full evaluation schema in addition to the attestation that we currently have in the evaluator app?

**Comment by bitbeckers** [comment]
Yes, potentially even lists of criteria or concattenated attestaions (c-atts). I thinks it's reasonable to note have this in scope now, as the repository of schema's is being rebuilt at the moment. But the Metrics Garden is a good source of metrics and schemas to pull from.

**Comment by holkexyz** [comment]
OK, but starting with a low number of schemas can also be useful for user not to get lost in them. I want to be very intentional about it. I think it is more a user question than a technical question.
